common/source/docs/common-hv-pm.rst

```diff
@@ -0,0 +1,30 @@
+.. _common-hv-pm:
+
+================================
+HV PM(High voltage power module)
+================================
+
+Overview：
+=========
+
+The CUAV HV_PM module is a new high voltage voltage power module independently developed by CUAV with the following features:
+* Higher voltage input: 10v-60v (3s~14s battery)
+* More accurate battery monitor: 
+  * voltage detection accuracy: +-0.1v
+  * current detection accuracy: +-0.4A
+* Bec (5v) max current: 5A
+* Max (detection) current: 60A
+* Max output current(ESC/MOTOR PORT): 60A
+
+Enable HV PM
+============
+
+.. image:: ../../../images/hv-pm.jpg
+:target: ../_images/hv-pm.jpg
+
+* Monitor：Anglog Voltage and Current
```

Comment:
Should provide params as above in previous uavcan pmu, not just rely on Mission Planner to set them

Reply:
Modified has been completed, please help me check it.